lib/EasyPost/Webhook.php

```diff
@@ -82,4 +84,34 @@ public static function create($params = null, $apiKey = null)
 
         return self::_create(get_class(), $params, $apiKey);
     }
+
+    /**
+     * Validate a webhook originated from EasyPost by comparing the HMAC header to a shared secret.
+     * This function will return True if the webhook originated from EasyPost and False if it did not.
+     *
+     * @param mixed $webhook_body
+     * @param mixed $headers
+     * @param string $webhook_secret
+     * @return boolean
+     */
+    public static function validateWebhook($webhook_body, $headers, $webhook_secret)
```

Comment:
Function params also need to be camel cased (the rest will get updated in the rewrite)

```suggestion
    public static function validateWebhook($webhookBody, $headers, $webhookSecret)
```

Reply:
It's still in progress, I will correct this